### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by casperlamboo**
These are the translations we got from lion bridge

CURA-8753

### CODE REVIEW

**Review by Ghostkeeper — CHANGES_REQUESTED** [review]

File: resources/i18n/zh_CN/cura.po [hunk 1]
```diff
@@ -5027,157 +5029,157 @@ msgstr "成功导出材料至： <filename>%1</filename>"
...
 msgctxt "@text"
 msgid "Click the export material archive button."
-msgstr ""
+msgstr "单击导出材料存档按钮。"
 
 #: /home/clamboo/Desktop/Cura/resources/qml/Preferences/Materials/MaterialsSyncDialog.qml:680
 msgctxt "@text"
 msgid "Save the .umm file on a USB stick."
-msgstr ""
+msgstr "将 .umm文件保存到 U 盘。"
```
Comment (casperlamboo) [inline resources/i18n/zh_CN/cura.po:5165]:
```suggestion
msgstr "将 .umm文件保存到 USB 盘。"
```

File: resources/i18n/zh_CN/cura.po [hunk 2]
```diff
@@ -5027,157 +5029,157 @@ msgstr "成功导出材料至： <filename>%1</filename>"
...
 
 #: /home/clamboo/Desktop/Cura/resources/qml/Preferences/Materials/MaterialsSyncDialog.qml:681
 msgctxt "@text"
 msgid "Insert the USB stick into your printer and launch the procedure to load new material profiles."
-msgstr ""
+msgstr "将 U 盘插入打印机，并启动程序以加载新材料配置文件。"
```
Comment (casperlamboo) [inline resources/i18n/zh_CN/cura.po:5170]:
```suggestion
msgstr "将 USB 盘插入打印机，并启动程序以加载新材料配置文件。"
```

File: resources/i18n/zh_CN/cura.po [hunk 3]
```diff
@@ -228,35 +228,35 @@ msgstr "找不到位置"
...
 msgctxt "@text:error"
 msgid "The response from Digital Factory is missing important information."
-msgstr ""
+msgstr "来自 Digital Factory 的响应缺少重要信息。"
 
 #: /home/clamboo/Desktop/Cura/cura/PrinterOutput/UploadMaterialsJob.py:218
 msgctxt "@text:error"
 msgid "Failed to connect to Digital Factory to sync materials with some of the printers."
-msgstr ""
+msgstr "无法通过 Digital Factory 同步材料与某些打印机。"
```
Comment (xli12) [inline resources/i18n/zh_CN/cura.po:254]: 无法通过 Digital Factory 为某些打印机同步材料配置文件。

File: resources/i18n/zh_CN/cura.po [hunk 4]
```diff
@@ -616,7 +616,7 @@ msgstr "所提供的状态不正确。"
 #: /home/clamboo/Desktop/Cura/cura/OAuth2/AuthorizationRequestHandler.py:80
 msgctxt "@message"
 msgid "Timeout when authenticating with the account server."
-msgstr ""
+msgstr "使用帐户服务器进行身份验证时超时。"
```
Comment (xli12) [inline resources/i18n/zh_CN/cura.po:619]: 使用帐户服务器进行身份验证超时。

File: resources/i18n/zh_CN/cura.po [hunk 5]
```diff
@@ -5027,157 +5027,157 @@ msgstr "成功导出材料至： <filename>%1</filename>"
...
-msgstr ""
+msgstr "要自动将材料配置文件与连接到 Digital Factory 的所有打印机同步，您需要登录 Cura。"
 
 #: /home/clamboo/Desktop/Cura/resources/qml/Preferences/Materials/MaterialsSyncDialog.qml:165
 #: /home/clamboo/Desktop/Cura/resources/qml/Preferences/Materials/MaterialsSyncDialog.qml:476
 #: /home/clamboo/Desktop/Cura/resources/qml/Preferences/Materials/MaterialsSyncDialog.qml:611
 msgctxt "@button"
 msgid "Sync materials with USB"
-msgstr ""
+msgstr "同步材料与 USB"
```
Comment (xli12) [inline resources/i18n/zh_CN/cura.po:5067]: 使用 USB 同步材料

File: resources/i18n/zh_CN/cura.po (hunk 1 above)
Comment (xli12) [reply resources/i18n/zh_CN/cura.po:5165]: Both U 盘 and USB 盘 are fine. The former one is a more colloquial form.

**Review by xli12 — COMMENTED** [review]
Added a few comments for simplified Chinese translation.

File: resources/i18n/zh_CN/cura.po (hunk 4 above)
Comment (casperlamboo) [inline resources/i18n/zh_CN/cura.po:619]:
```suggestion
msgstr "使用帐户服务器进行身份验证超时。"
```

File: resources/i18n/zh_CN/cura.po (hunk 3 above)
Comment (casperlamboo) [inline resources/i18n/zh_CN/cura.po:254]:
```suggestion
msgstr "无法通过 Digital Factory 为某些打印机同步材料配置文件。"
```

File: resources/i18n/zh_CN/cura.po (hunk 5 above)
Comment (casperlamboo) [inline resources/i18n/zh_CN/cura.po:5067]:
```suggestion
msgstr "使用 USB 同步材料"
```

File: resources/i18n/zh_CN/cura.po (hunk 2 above)
Comment (Ghostkeeper) [reply resources/i18n/zh_CN/cura.po:5170]: Same as above, we'll leave it as `U 盘`.

**Review by Ghostkeeper — APPROVED** [review]

**Pull request merged**